### ISSUE TITLE
how to get class label values & confusion matrix at test ? its not sufficient to get only accuracy..!

### ISSUE THREAD

**Issue by amitchougule121** [HRNet/HRNet-Image-Classification#17]
I try with "HRNet-Image-Classification/lib/core/function

[function.txt](https://github.com/HRNet/HRNet-Image-Classification/files/3955217/function.txt)

.py" in this file, at line no. 104,

```
for i, (input, target) in enumerate(val_loader):
            output = model(input)
            batch_time.update(time.time() - end)
            target = target.cuda(non_blocking=True)
            loss = criterion(output, target)
```
in this I try with target variable, it shows correct values at train time, but at test time it shows the class label for image in which its stored. not accurate values. can anyone tell how to get class label values for "test.py" ? target variable shows inaccurate values at testing.
